daemon/src/payout_curve/csr_tools.rs

```diff
@@ -149,4 +148,61 @@ mod tests {
 
         assert!(matches!(a, Error::CannotInitCSR));
     }
+
+    // test that all is good
+    #[test]
+    fn test_lu_matrix_solve_00() {
+        let a = CSR::new(
+            Array1::<f64>::from_vec(vec![11., 12., 22., 23., 31., 33.]),
```

Comment:
The parameter type in `new` enforces the type of the array (`f64`), hence you wouldn't need to specify it here. That would reduce some noise in these tests.